stories/Link.stories.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import Link from 'components/generic/Link';
+
+import Provider from './Provider';
+import configureStore from './configuedStore.js';
+
+import { foundationPartial } from 'styles/foundation';
+import ebiGlobalStyles from 'ebi-framework/css/ebi-global.css';
+
+const f = foundationPartial(ebiGlobalStyles);
+
+const store = configureStore();
+
+const withProvider = (story) => <Provider store={store}>{story()}</Provider>;
+
+export default {
+  title: 'InterPro UI/Link',
+  decorators: [withProvider],
+};
+
+const initialLocation = {
+  description: {
+    entry: {
+      db: 'InterPro',
+      accession: 'IPR000001',
+    },
+    main: {
+      key: 'entry',
+    },
+  },
+  search: {},
+};
+
+export const Basic = () => (
+  <>
+    Visit
+    <Link href="https://www.ebi.ac.uk/interpro"> Interpro</Link> /
+    <Link href="https://www.ebi.ac.uk/" className={f('ext')} target={'_blank'}>
+      {' '}
+      EMBL-EBI
+    </Link>
+  </>
+);
+
+export const InternalLink = () => {
+  // TODO yet to be fixed
```

Comment:
The way I did it in the MemberDBSelector story was to import the component without the connect so instead of importing `Link` import `_Link` (you gonna have to expose that in the Link component.)
That way you can pass the `setLocation` as a mock for `goToCustomLocation`

Reply:
Your code remind me of the mocked redux store that is been using in connected table, so I created a ConnectedLink using that strategy.
Also added a `<pre>` area showing the current state of the location